app/models/donation_chart.rb

```diff
@@ -13,20 +13,25 @@ def self.donation_issues
   end
 
   def self.wtf
-    Issue.all.map do |issue|
-      issue.candidate_issues.map { |candidate_issue| candidate_issue.donations.
-        map { |donation| donation.amount }}.reduce(:+).reduce(:+)
-    end.map do |num|
-      if num.nil?
-        0
-      else
-        num
+    omg = Issue.all.map do |issue|
+        issue.candidate_issues.map { |candidate_issue| candidate_issue.donations.
+          map { |donation| donation.amount }}
```

Comment:
Expression at 18, 45 should be on its own line.<br>Align the operands of an expression in an assignment spanning multiple lines.<br>Pass `&:amount` as an argument to `map` instead of a block.